lib/options.py

```diff
@@ -252,6 +260,11 @@ def __init__(self):
                 action="store_true",
                 default=False,
                 help="""Disable schema upgrade on testing with snapshots.""")
+        parser.add_argument(
+                "--allocator",
+                dest="allocator",
+                default=env_string('TEST_RUN_MEMTX_ALLOCATOR', "small"),
```

Comment:
The name of the test-run parameter (`--allocator`) is not consistent with the name of test-run's environment variable (`TEST_RUN_MEMTX_ALLOCATOR`). The name of the environment variable is not consistent with the name of the box.cfg parameter (`allocator`). I would use the same name (ignoring lower/upper case, dash/underscore, `--` or `TEST_RUN_` prefix) everywhere. Is there decision how the `box.cfg` will be named: `allocator` or `memtx_allocator`?

Reply:
memtx_allocator

Reply:
Now the names are consistent. Thanks!